packages/contentful/migrations/crn/labs/20250202113054-rollback-lab-membership-and-adds-pi-field.js

```diff
@@ -0,0 +1,47 @@
+module.exports.description =
+  'Renames lab membership reference (labs to oldLabs) and creates new labs field (references labs) adds PI field';
+
+module.exports.up = (migration) => {
+  const labs = migration.editContentType('labs');
+
+  labs
+    .createField('labPI')
+    .name('Lab PI')
```

Comment:
Maybe the display name could be

```suggestion
    .name('Principle Investigator')
```

as it appears in the ticket 
<img width="1109" alt="Screenshot 2025-02-11 at 09 32 56" src="https://github.com/user-attachments/assets/5fe74b4e-301e-4da0-9c29-0ccf4dbe9eaa" />